detection_rules/rule.py

```diff
@@ -7,19 +7,24 @@
 from dataclasses import dataclass, field
 from functools import cached_property
 from pathlib import Path
-from typing import Literal, Union, Optional, List, Any
+from typing import Literal, Union, Optional, List, Any, Dict
 from uuid import uuid4
 
 from marshmallow import ValidationError, validates_schema
 
-from . import utils
+import eql
+import kql
+
+from . import beats, ecs, utils
 from .mixins import MarshmallowDataclassMixin
 from .rule_formatter import toml_write, nested_normalize
 from .schemas import definitions
-from .schemas import downgrade
-from .utils import cached
+from .schemas import get_stack_schemas, downgrade
+from .utils import cached, load_etc_dump
+
 
 _META_SCHEMA_REQ_DEFAULTS = {}
+CURRENT_PACKAGE = load_etc_dump('packages.yml')['package']['name']
```

Comment:
I think we do this in several places now, so should be able to leverage that.
Also, I generally prefer to not make constants on file read + deserialize. A YML error causing an import error can be confusing.